README.md

```diff
@@ -20,6 +20,22 @@ The GitHub Actions workflow is scheduled to run once a day to:
 2. wait until each export is done
 3. download, unzip and commit the content from each export to the repository
 
+## Notion Backup Enhancer
```

Comment:
this section should be more like "Similar Projects" or "Related" or "Other Notion Backup Projects" or "See Also" or something like this

check this example:

- https://github.com/paulirish/lite-youtube-embed#other-fast-youtube-embeds

the content of the section should be 1 single bullet point with a title, short description and URL of your project, no paragraphs or screenshots